client/client.go

```diff
@@ -59,8 +104,15 @@ func (client *Client) Download(path string) error {
 	if err != nil {
 		return err
 	}
-	file := make([]byte, client.File.PieceLength)
```

Comment:
it is tested?

Reply:
The test coverage for this PR is not pretty. I will add some tests.